main.js

```diff
@@ -12,16 +12,31 @@ var ideaBody = document.querySelector(".idea-body");
 var formTitle = document.querySelector("#title-form");
 var formBody = document.querySelector("#body-form");
 var formSearch = document.querySelector("#search-form");
-
+var ideaCards = document.querySelector(".idea-cards");
 //event listeners:
 saveButton.addEventListener("click", saveIdea);
 
 //functions:
+
+
+
 function saveIdea(event) {
     event.preventDefault();
     var newCard = new Ideas(formTitle.value, formBody.value);
-    ideaTitle.innerText = newCard.title;
-    ideaBody.innerText = newCard.body;
+    ideaCards.innerHTML += `<section class="card">
+    <div class="top-bar">
+      <button class="star-btn" type="submit"><img src="assets/star-active.svg" alt="star button"></button>
+      <button class="delete-btn" type="submit"><img src="assets/delete-active.svg" alt="delete button"></button>
+    </div>
+    <div class="body-section">
+      <h3 class="idea-title">${formTitle.value}</h3>
+      <p class="idea-body">${formBody.value}</p>
+    </div>
+    <div class="bottom-bar">
+      <button class="comment-btn" type="submit"><img src="assets/comment.svg" alt="comment button"></button>
+      <label for="comment-btn">Comment</label>
+    </div>
+  </section>`
```

Comment:
This looks excellent! But I'm wondering if it actually shouldn't maybe be a different function?